src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/dashboard/fix/data-broker-profiles/removal-under-maintenance/RemovalUnderMaintenanceView.tsx

```diff
@@ -0,0 +1,219 @@
+      </p>
+      <p>{l10n.getString("data-broker-removal-guide-top-section-para-1")}</p>
+      <br />
+      <p>{l10n.getString("data-broker-removal-guide-top-section-para-2")}</p>
+
+      <div className={styles.removalContentSection}>
+        <dt className={styles.removalGuideInstructions}>
+          {l10n.getString("data-broker-removal-guide-step-1-header")}
+        </dt>
+        <dd>{l10n.getString("data-broker-removal-guide-step-1-body")}</dd>
+        <ul>
+          <li>
+            {l10n.getString("data-broker-removal-guide-step-1-list-item-1")}
+          </li>
+          <li>
+            {l10n.getString("data-broker-removal-guide-step-1-list-item-2")}
+          </li>
+          <li>
+            {l10n.getString("data-broker-removal-guide-step-1-list-item-3")}
+          </li>
+          <li>
+            {l10n.getString("data-broker-removal-guide-step-1-list-item-4")}
+          </li>
+        </ul>
+      </div>
+
+      <div className={styles.removalContentSection}>
+        <dt className={styles.removalGuideInstructions}>
+          {l10n.getString("data-broker-removal-guide-step-2-header")}
+        </dt>
+        <dd>
+          {l10n.getString("data-broker-removal-guide-step-2-body-para-1")}
+        </dd>
+        <dd>
+          {l10n.getString("data-broker-removal-guide-step-2-body-para-2")}
+        </dd>
+      </div>
+
+      <div className={styles.removalContentSection}>
+        <dt className={styles.removalGuideInstructions}>
+          {l10n.getString("data-broker-removal-guide-step-3-header")}
+        </dt>
+        <dd>
+          {l10n.getString("data-broker-removal-guide-step-3-body-para-1")}
+        </dd>
+        <dd>
+          {l10n.getString("data-broker-removal-guide-step-3-body-para-2")}
+        </dd>
+      </div>
+
+      <div className={styles.removalContentSection}>
+        <dt className={styles.removalGuideInstructions}>
+          {l10n.getString("data-broker-removal-guide-step-4-header")}
+        </dt>
+        <dd>{l10n.getString("data-broker-removal-guide-step-4-body")}</dd>
+      </div>
+
+      <div className={styles.buttonWrapper}>
+        <TelemetryButton
+          onPress={() => setDetailedRemovalGuide(!detailedRemovalGuide)}
+          variant="primary"
+          event={{
+            module: "button",
+            name: "click",
+            data: {
+              button_id: "return_to_data_broker",
+            },
+          }}
+        >
+          {l10n.getString("data-broker-removal-guide-button-back-to-exposures")}
+        </TelemetryButton>
+      </div>
+    </div>
+  );
+
+  return (
+    <FixView
+      subscriberEmails={props.subscriberEmails}
+      nextStep={getNextGuidedStep(props.stepDeterminationData, "Scan")}
+      currentSection="data-broker-profiles"
+      data={props.stepDeterminationData}
+      hideProgressIndicator={detailedRemovalGuide}
+    >
+      {!detailedRemovalGuide ? dataBrokerInformation : removalGuideInstructions}
```

Comment:
The only way to go back to the exposures when the “removal instructions” are open is by clicking “Back to exposures” (which for many viewport sizes is out of sight). I’m wondering if it would not be more intuitive to follow our existing pattern of showing explainers in dialogs. If that was already decided against by UX and/or is out of scope we should at least make sure that the closing “x” button does not link back to the dashboard.

<img width="1002" alt="image" src="https://github.com/user-attachments/assets/9ea2148f-636c-4869-996c-f3d7bf84e9ed">